src/main/java/school/hei/haapi/model/Grade.java

```diff
@@ -0,0 +1,58 @@
+package school.hei.haapi.model;
+
+import java.io.Serializable;
+import java.time.Instant;
+import java.util.Objects;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import org.hibernate.Hibernate;
+
+@Entity
+@Table(name = "\"grade\"")
+@Getter
+@Setter
+@ToString
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class Grade implements Serializable {
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private String id;
+
+  @ManyToOne
+  @JoinColumn(name = "student_course_id")
+  private StudentCourse studentCourse;
+  @ManyToOne
+  @JoinColumn(name = "exam_id")
+  private Exam exam;
+  private int score;
```

Comment:
Be careful when using primitive type, because sometimes, column type handle null value and it could directly set NPE (NullPointer Exception)